package-generator/settings/cookbooks_urls

```diff
@@ -1,21 +1,23 @@
 [main]
 java: https://github.com/agileorbit-cookbooks/java.git
 openssl: https://github.com/chef-cookbooks/openssl.git
-build-essential: https://github.com/chef-cookbooks/build-essential.git
+build-essential: https://forge.fiware.org/scmrepos/svn/testbed/trunk/cookbooks/BaseRecipes/build-essential/
 apt: https://forge.fiware.org/scmrepos/svn/testbed/trunk/cookbooks/BaseRecipes/apt-spagobi/
 git: https://github.com/chef-cookbooks/git.git
 yum-mysql-community: https://github.com/chef-cookbooks/yum-mysql-community.git
 yum-epel: https://github.com/chef-cookbooks/yum-epel.git
 apache2: https://github.com/svanzoest-cookbooks/apache2.git
 php: https://github.com/chef-cookbooks/php.git
-mysql: https://github.com/chef-cookbooks/mysql.git
+mysql: https://forge.fiware.org/scmrepos/svn/testbed/trunk/cookbooks/BaseSoftware/mysql/
```

Comment:
Why do you need to get these products from the forge and not from the chef-cookbooks repository?

Idem for poise...